src/SegmentedControl/SegmentedControl.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import lodash from "lodash";
+
+import "./SegmentedControl.less";
+
+/*
+  SegmentedControl selection. Shows several options
+  as defined in the props and allows the user to select
+  one of those options.
+*/
+export class SegmentedControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {selected: props.defaultValue};
```

Comment:
i think `defaultValue` and `value` are appropriate here - similar to what react uses for the `<select>` element.

`defaultValue` should be applied if the control's state hasn't been modified by the actual user and `value` should always override the visual state, regardless of what the user previously selected.

`value` would have to be used with `onChange` to keep the control's state in sync with the parent (i think react locks the control if a `value` is specified without an `onChange`

simpler to just support `defaultValue`, IMO, since `value` is more involved - can always add that later if there's a need